lookout/style/format/features.py

```diff
@@ -225,17 +247,95 @@ def _classify_vnodes(self, nodes: Iterable[VirtualNode]) -> List[VirtualNode]:
                             y=cls))
         return result
 
+    def _inplace_write_vnode_features(self, features: numpy.ndarray, vnode: VirtualNode, row: int,
```

Comment:
The parameters order should be first input then output (e.g. `_inplace_write_vnodes_features`)